src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -466,9 +507,23 @@ else if (SiteUtil.JAVADOC_TOKENS.equals(propertyName)) {
         else {
             final String defaultValue = SiteUtil.getDefaultValue(
                     propertyName, field, instance, currentModuleName);
-            sink.rawText(CODE_START);
-            sink.text(defaultValue);
-            sink.rawText(CODE_END);
+
+            final String checkName = CHECK_PATTERN
+                    .matcher(instance.getClass().getSimpleName()).replaceAll("");
+
+            final boolean isSpecialTokenProp = NON_BASE_TOKEN_PROPERTIES.stream()
+                    .anyMatch(tokenProp -> tokenProp.equals(checkName + " - " + propertyName));
+
+            if (isSpecialTokenProp && !CURLY_BRACKET.equals(defaultValue)) {
+                final List<String> defaultValuesList =
+                        Arrays.asList(COMMA_SPACE_PATTERN.split(defaultValue));
+                writeTokensListDotLess(sink, defaultValuesList, SiteUtil.PATH_TO_TOKEN_TYPES);
```

Comment:
````suggestion
                writeTokensList(sink, defaultValuesList, SiteUtil.PATH_TO_TOKEN_TYPES, false);
````

Add a boolean onto the end of `writeTokensList`, and use that to control the period. Other `writeTokensList` calls should use `true`. Remove `writeTokensListDotLess`.

Reply:
done.

Reply:
@rnveach, the idea of a introducing a boolean variable was insightful. thank you @romani for the help.

Reply:
@relentless-pursuit I am against duplicating chunks of code if we can achieve what we need with simple changes.